docroot/modules/custom/va_gov_backend/js/service_location_address.es6.js

```diff
@@ -5,6 +5,24 @@
 ((Drupal) => {
   Drupal.behaviors.vaGovServiceLocationAddress = {
     attach(context) {
+      // Add or remove the classes and attributes needed to make the address required.
+      function addressRequired(address, action) {
```

Comment:
This isn't critical, just something to have in your back pocket for the future, @JayDarnell.  We are using es6, so we have access to the newer arrow function syntax.
```suggestion
      const addressRequired = (address, action) => {
```